src/main/java/net/sf/jabref/gui/maintable/MainTable.java

```diff
@@ -499,11 +499,14 @@ public void setSelected(int row) {
     }
 
     public int findEntry(BibEntry entry) {
-        return model.getTableRows().indexOf(entry);
-    }
-
-    public int findLastEntry(BibEntry entry) {
-        return model.getTableRows().lastIndexOf(entry);
+        EventList<BibEntry> tableRows = model.getTableRows();
```

Comment:
I think in this particular case it is fine to use an "old school" for loop. But please rename `i` to `row`.